services/web/server/src/simcore_service_webserver/workspaces/_workspaces_db.py

```diff
@@ -65,11 +72,11 @@ async def create_workspace(
             )
             .returning(*_SELECTION_ARGS)
         )
-        row = await result.first()
+        row = result.first()
```

Comment:
And here. I would expect this one to be await maybe the `execute` above should be a `stream`?